### DISCUSSION TITLE
MDS verification fails b/c AAGUID isn't in blob

### DISCUSSION THREAD

**Discussion by zfLQ2qx2** [go-webauthn/webauthn#340] [General]
So I'm using two webauthn devices to test - Yubikey 5 and 1Password. No issues with Yubikey 5, I have that working for registration and login. However 1Password is working for registration but not for Login:

```Failed to validate credential record metadata (invalid_request): Error occurred validating authenticator metadata from the credential record: error occurred performing authenticator entry validation: AAGUID entry has not been registered with the metadata service```

The AAGUID for 1Password is "bada5566-a7aa-401f-bd96-45619a55120d" (love their developer has a sense of humor - bada55) and it is not in the MDS blob. Looking at the code the only way I see past this is to not initialize the metadata provider at all.

But in a production environment where I can't control what fido devices people bring/use, what should I do in situations like this?

**Comment by james-d-elliott** [comment]
https://github.com/go-webauthn/webauthn/blob/master/metadata/types.go#L19-L21

If this doesn't work then it's likely a bug.
